src/Network/MPD/Applicative/PlaybackControl.hs

```diff
@@ -57,6 +58,13 @@ seek pos time = Command emptyResponse ["seek" <@> pos <++> time]
 seekId :: Id -> FractionalSeconds -> Command ()
 seekId id' time = Command emptyResponse ["seekid" <@> id' <++> time]
 
+-- | Seek to time in the current song. Absolute time for True in
```

Comment:
Can you add the haddock since annotation for this and the other functions which has been exported ? Thanks!

Reply:
Done.